Multiprotocol/FX_nrf24l01.ino

```diff
@@ -33,29 +33,62 @@ Multiprotocol is distributed in the hope that it will be useful,
 #define FX620_PAYLOAD_SIZE			7
 #define FX620_CH_OFFSET				1
 
+#define FX9630_PACKET_PERIOD		8124
+#define FX9630_BIND_PACKET_PERIOD	8124
+#define FX9630_BIND_CHANNEL			51
+#define FX9630_PAYLOAD_SIZE			8
+#define FX9630_NUM_CHANNELS			3
+
 //#define FORCE_FX620_ID
+#define FORCE_FX9630_ID
 
 static void __attribute__((unused)) FX_send_packet()
 {
 	//Hopp
 	if(IS_BIND_DONE)
 	{
-		XN297_Hopping(hopping_frequency_no++);
-		hopping_frequency_no &= 0x03;
+		if(sub_protocol == FX9630)
+		{
+			XN297_SetTXAddr(rx_tx_addr, 4);
+			XN297_Hopping(hopping_frequency_no++);
+			if (hopping_frequency_no > FX9630_NUM_CHANNELS)
+				hopping_frequency_no = 0;
+		}
+		else // FX816 and FX620
+		{
+			XN297_Hopping(hopping_frequency_no++);
+			hopping_frequency_no &= 0x03;
+		}
 	}
 
 	memset(packet,0x00,packet_length);
 
 	//Channels
-	uint8_t offset=sub_protocol == FX816 ? FX816_CH_OFFSET:FX620_CH_OFFSET;
-	uint8_t val=convert_channel_8b(AILERON);
-	if(val>127+FX_SWITCH)
-		packet[offset] = sub_protocol == FX816 ? 1:0xFF;
-	else if(val<127-FX_SWITCH)
-		packet[offset] = sub_protocol == FX816 ? 2:0x00;
-	else
-		packet[offset] = sub_protocol == FX816 ? 0:0x7F;
-	packet[offset+1] = convert_channel_16b_limit(THROTTLE,0,100);	//FX816:0x00..0x63, FX620:0x00..0x5E but that should work
+	uint8_t val;
+	if (sub_protocol == FX9630)
+	{
+		packet[0] = convert_channel_8b(THROTTLE);
+		packet[1] = convert_channel_8b(AILERON);
+		packet[2] = 0xFF - convert_channel_8b(ELEVATOR);
+		packet[3] = convert_channel_8b(RUDDER);
+		packet[4] = 0x20;
+    val = convert_channel_8b(5);
+    packet[5] = val > 127 ? 0x01 : 0x00;  // DR toggle swich: 0 small throw, 1 large throw
+    val = convert_channel_8b(6);
```

Comment:
Use the direct values instead:
packet[5] |= (Channel_data[CH6] < CHANNEL_MIN_COMMAND ? (Channel_data[CH6] > CHANNEL_MAX_COMMAND ? 0x02 : 0x01) : 0x00) << 1;  // Mode A(0) : 6D small throw, B(1) : 6D large throw, C(2) : 3D